### DISCUSSION TITLE
Remaining distros shipping vm86-enabled kernels?

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [dosemu2/dosemu2#2260] [Q&A]
So Debian i386 gives:
```
ERROR: vm86 service not available in your kernel, Functie is niet aanwezig
```

Debian's Linux builds are not including vm86 anymore. Presumably since its support was disabled by default upstream.

Does anybody know if anybody is making a distro or Linux build available that does have vm86 enabled?

Would it likely have a significant impact on performance?

**Comment by stsp** [comment]
No because KVM is to the rescue.

**Reply by jschwartzenberg** [reply]
Yeah, but if the CPU would support KVM, I would not even install a i386 distro in the first place....

The system I'm trying this on has a Celeron M 410.
